includes/User.php

```diff
@@ -2685,6 +2685,14 @@ public function isAnon() {
 	}
 
 	/**
+	 * Whether this user is Wikia staff or not
+	 * @return bool
+	 */
+	public function isStaff() {
+		return in_array( 'staff', $this->getEffectiveGroups() );
```

Comment:
This starts to pull the thread to a lot of code ... I'm not sure I want to expand this PR for that many potential changes ;)